src/App.js

```diff
@@ -57,13 +57,13 @@ class App extends Component {
     return (
       <div className="game">
         <section className="hud">
-          <h2 className="score">Litter Spotted: { this.state.points }</h2>
+          <h2 className="score">Litter Spotted: {this.state.points}</h2>
           <img className="logo" src={logo} alt="Litter Patrol logo" />
         </section>
 
         <section className="level">
-          { this.levelBackground() }
-          { items }
+          {this.levelBackground()}
+          {items}
```

Comment:
Here you remove space around curly braces, but below you add it. You should pick one.

Reply:
There's no space around JSX interpolation, but space around JS objects.